content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -285,6 +285,29 @@ If you restrict `list` or `watch` by resourceName, clients must include a `metad
 For example, `kubectl get configmaps --field-selector=metadata.name=my-configmap`
 {{< /note >}}
 
+Rather than referring to individual `resources` and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
+For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `apiGroups` and `resourceNames` an empty set means that everything is allowed.
+Here is an example that allows access to perform any current and future action on all current and future resources (note, this is similar to the built-in `cluster-admin` role).
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: very-open-role  # DO NOT USE THIS ROLE, IT IS JUST AN EXAMPLE
+rules:
+- apiGroups: [""]
```

Comment:
to make this example role less dangerous, I'd suggest scoping it to an example API group, so it wouldn't actually grant access to real resources in practice:

```yaml
- apiGroups: ["example.com"]
  resources: ["*"]
  verbs: ["*"]
```

Reply:
good idea - done in https://github.com/kubernetes/website/pull/36043/commits/3de0d5b47d44f524c5de912bd3b4e7e5082b8c45